### DISCUSSION TITLE
RoadMap?

### DISCUSSION THREAD

**Discussion by FallenDoll** [WorldObservationLog/Akiwa#2] [Q&A] [answered]
Hello, I would like to know if your repo has a roadmap, so that I can understand your plans and goals. If so, could you please share the link or file of the roadmap? If not, do you intend to create one? I think this would be very helpful for the project management and promotion. Thank you for your reply.🤗

**Comment by WorldObservationLog** [comment, marked as answer]
Currently Akiwa is only a personal project and has not any further plans. We need more feedback and suggestions to help complete the project. But you can check [Akiwa V1.0 Kanban](https://github.com/users/WorldObservationLog/projects/2) for some infomations.
